requirements/edx/custom.txt

```diff
@@ -11,7 +11,7 @@
 # This is currently using a custom branch: https://github.com/open-craft/xblock-poll/tree/copy-change-patch
 -e git+https://github.com/open-craft/xblock-poll.git@3a06ef4782f9f90d6a8594c4eb4a84a277182fb8#egg=xblock-poll
 -e git+https://github.com/edx/edx-notifications.git@0.6.0#egg=edx-notifications==0.6.0
--e git+https://github.com/open-craft/problem-builder.git@fa1bcb6ac4216beda6c96b0d8539e87cea805f7e#egg=xblock-problem-builder
+-e git+https://github.com/open-craft/problem-builder.git@2.7.0#egg=xblock-problem-builder==2.7.0
```

Comment:
@e-kolpakov  Looks like the tag is `v2.7.0` not `2.7.0`, which is why CircleCI failed.